x/ecocredit/client/marketplace/query.go

```diff
@@ -0,0 +1,242 @@
+package marketplace
+
+import (
+	"strconv"
+
+	sdkclient "github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+
+	"github.com/regen-network/regen-ledger/x/ecocredit"
+	"github.com/regen-network/regen-ledger/x/ecocredit/marketplace"
+)
+
+// QuerySellOrderCmd returns a query command that retrieves information for a given sell order.
+func QuerySellOrderCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "sell-order [sell_order_id]",
+		Short: "Retrieve information for a given sell order",
```

Comment:
We should probably include `Long` descriptions with examples. I was thinking about doing this for all commands as part of #931 but might be nice to get started with marketplace commands here.